pages/common/wp.md

```diff
@@ -11,6 +11,14 @@
 
 `wp cli update`
 
+- Download a fresh WordPress installation to current directory:
+
+`wp core download [--locale=<locale>]`
+
+- Create basic wpconfig file:
+
+`wp config create --dbname=<dbname> --dbuser=<dbuser> [--dbpass=<dbpass>] [--dbhost=<dbhost>]`
```

Comment:
As already said, we can't use the `[ ]` syntax from man pages. Are those last two flags useful? Either we write them without it or we remove them if they're not that important.

Reply:
I'd guess the --dbpass is important enough to specify it, --dbhost however probably isn't.

Reply:
These are very common DB arguments, I think we can left all.